### ISSUE TITLE
Add "Wallet only mode"

### ISSUE THREAD

**Issue by Swader** [status-im/status-react#7729]
# Problem
Sometimes, I just want to use a dapp or send / receive assets. I don't care about chat at that moment - maybe I'm on a limited connection, maybe others in my household are using the internet for streaming and fetching messages would kill this, maybe I literally don't need chat, I just want to send some eth. Status starts syncing chat by default, which is detrimental to UX in such use cases.

# How to solve
Introduce a "start in:" selector to options.
Options available:

* wallet only mode
* chat + wallet mode

When Status is in wallet only mode, make chat refresh a manual process by either long-pressing on chats, or by toggling a settings switch, or something else.

In a nutshell, allow for a wallet-only mode of Status.

- - -
Note this would also disable chat under dapps. An acceptable compromise.

**Comment by 0xc1c4da** [comment]
➤ Rachel Hamlin commented:

Did this come up in context of v1 launch Nabil Naghdy?

**Comment by 0xc1c4da** [comment]
➤ Nabil Naghdy commented:

No it didn't. I just moved it, because I was playing around with Wrike :)

**Comment by 0xc1c4da** [comment]
➤ Rachel Hamlin commented:

Okay, gotcha!

**Comment by 0xc1c4da** [comment]
➤ Nabil Naghdy commented:

Yeah. Feel free to move to a future work folder

**Comment by cammellos** [comment]
Closing this for now as will not be implemented any time soon, feel free to reopen with more details.
